docs/usage.md

```diff
@@ -104,6 +107,28 @@ Sample files that can be used as illustrative references can be found in this re
 | last_name | str | User's last name |
 | email | str | User's email |
 
+## `ib_input_instances.csv`
+
+| Column | Type | Notes |
+| - | - | - |
+| id | UUID | The value of the data-content-id |
+| content_id | UUID | Content page UUID |
+| variant | str | Variant name |
+| content | str | Input content |
+| prompt | str | Prompt content |
+
+## `ib_pset_problems.csv`
+
+| Column | Type | Notes |
+| - | - | - |
+| id | UUID | The value of the data-content-id |
+| content_id | UUID |  Content page UUID |
+| variant | str | Variant name |
+| content | str | Problem content |
+| problem_type | str | Problem type |
```

Comment:
It might be nice to include the expected values: `Problem type (one of 'input', 'dropdown', 'multiselect', or 'multiplechoice')`